packages/frontend/src/components/rich_text_editor/commands.ts

```diff
@@ -69,7 +69,7 @@ export const increaseListIndet: Command = (state, dispatch, view) => {
     const { $from } = state.selection;
     let listNode = null;
     for (let i = $from.depth; i > 0; i--) {
-        if ($from.node(i).type.name === schema.nodes.list_item.name) {
+        if ($from.node(i).type.name === schema.nodes.list_item!.name) {
```

Comment:
Not sure why all these properties have types suggesting that they can be undefined. Of course, the bangs make the linter angry.